logdb/logdb.go

```diff
@@ -204,28 +196,21 @@ FROM (%v) t
 		subQuery += ")"
 	}
 
+	if filter.Order == DESC {
```

Comment:
does it make sense to extract it ? as same is above

Reply:
This commit is just reverting the code, I don't want to apply any different logic in case it affects performance